fuse/lag_node.go

```diff
@@ -0,0 +1,74 @@
+package fuse
+
+import (
+	"context"
+	"strconv"
+	"syscall"
+
+	"bazil.org/fuse"
+	"bazil.org/fuse/fs"
+)
+
+const LagFilename = ".lag"
+
+var _ fs.Node = (*LagNode)(nil)
+var _ fs.NodeForgetter = (*LagNode)(nil)
+var _ fs.HandleReadAller = (*LagNode)(nil)
+var _ fs.NodeListxattrer = (*PosNode)(nil)
+var _ fs.NodeGetxattrer = (*PosNode)(nil)
+var _ fs.NodeSetxattrer = (*PosNode)(nil)
+var _ fs.NodeRemovexattrer = (*PosNode)(nil)
+var _ fs.NodePoller = (*PosNode)(nil)
+
+type LagNode struct {
+	fsys *FileSystem
+}
+
+func newLagNode(fsys *FileSystem) *LagNode {
+	return &LagNode{fsys: fsys}
+}
+
+func (n *LagNode) Attr(ctx context.Context, attr *fuse.Attr) error {
+	attr.Mode = 0o444
+	attr.Size = uint64(len(n.content()))
+	attr.Uid = uint32(n.fsys.Uid)
+	attr.Gid = uint32(n.fsys.Gid)
```

Comment:
Can you set the mtime on this file based on the `PrimaryTimestamp()`? That'd make it so you could fetch the lag with only a `stat()` call.

Also, you'll need to set `attr.Valid` to `0` to avoid caching.